requirements.txt

```diff
@@ -8,7 +8,7 @@ tqdm
 gsutil
 
 # generate_sharktank
-transformers==4.18.0
+transformers==4.19.4
```

Comment:
Actually I only did it for the importer. I didn't realize we added transformers to the base requirements.txt @monorimet